src/pages/home/report/comment/TextCommentFragment.tsx

```diff
@@ -52,7 +52,10 @@ function TextCommentFragment({fragment, styleAsDeleted, source, style, displayAs
     // Only render HTML if we have html in the fragment
     if (!differByLineBreaksOnly) {
         const editedTag = fragment.isEdited ? `<edited ${styleAsDeleted ? 'deleted' : ''}></edited>` : '';
-        const htmlContent = styleAsDeleted ? `<del>${html}</del>` : html;
+        const htmlWithDeletedTag = styleAsDeleted ? `<del>${html}</del>` : html;
+
+        const containsOnlyEmojis = EmojiUtils.containsOnlyEmojis(text);
```

Comment:
I think that we can ignore this for now. I am creating a new issue to prevent markdown from affecting emojis as a whole, so this will be fixed in the future in a different way